google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ConditionalRowMutation.java

```diff
@@ -47,13 +53,23 @@ public static ConditionalRowMutation create(String tableId, String rowKey) {
     return create(tableId, ByteString.copyFromUtf8(rowKey));
```

Comment:
Here and other methods in this class, we can convert tableId to TargetId here. Annotate it with `@Deprecated` and add some doc.